tests/qos/qos_sai_base.py

```diff
@@ -1835,3 +1835,80 @@ def dualtor_ports_for_duts(request, get_src_dst_asic_and_duts):
         logger.info("Finish fetching dual ToR info {}".format(dualtor_ports_set))
 
         return dualtor_ports_set
+
+    @pytest.fixture(scope='function', autouse=True)
+    def set_static_route(
+            self, get_src_dst_asic_and_duts, dutTestParams, dutConfig):
+        # Get portchannels.
+        # find the one that is backplane based.
+        # set a static route through that portchannel.
+        # remove when done.
+        if dutTestParams["basicParams"]["sonic_asic_type"] != "cisco-8000":
```

Comment:
check for switch_type as chassis-packet and topo as t2. Not asic type. this will make change more generic,

Reply:
@abdosi , if the switch_type is not provided in the ansible/lab file, should this code be executed?

Reply:
@abdosi: Fixed, but need to know what to do if the switch_type is not available.